main/reference/ertp-api/amount-math.md

```diff
@@ -113,12 +57,14 @@ const quatloos123 = AmountMath.make(quatloosBrand, 123n);
 const myValue = AmountMath.getValue(quatloosBrand, quatloos123);
 ```
 ## AmountMath.makeEmpty(brand, assetKind)
-- Returns: `{Amount}`
+- **brand** **[Brand](./brand.md)**
+- **assetKind** **[AssetKind](./ertp-data-types.md#assetkind)**
+- Returns: **[Amount](./ertp-data-types.md#amount)**
 
-Returns the `amount` representing an empty `Amount` for the `brand` argument's 
-`Brand`. This is the identity element for `AmountMath.add()` 
-and `AmountMath.subtract()`. The empty `value` depends 
-on whether the `assetKind` is `AssetKind.NAT` (`0n`) or `AssetKind.SET` (`[]`).
+Returns the **Amount** representing an empty **Amount** for the *brand* parameter's 
+**Brand**. This is the identity element for **AmountMath.add()** 
+and **AmountMath.subtract()**. The empty **Value** depends 
+on whether the **assetKind** is **AssetKind.NAT** (**0n**) or **AssetKind.SET** (**[]**).
```

Comment:
`AssetKind.SET` is deprecated. We should be talking about `AssetKind.CopySet` and `AssetKind.CopyBag`. I think we can't make `SET` go away yet.